sdks/python/apache_beam/io/gcp/bigtableio_it_test.py

```diff
@@ -26,6 +26,7 @@
 import uuid
 
 import pytz
+from nose.plugins.attrib import attr
```

Comment:
I think line 68 meant to call superclass of `beam.PTransform` (a copy-paste error from `WriteToBigTable` implementation?). This error is caught in Py3 test suites.
It seems strange that we need to create a new transform to write test data - why is that the case, @chamikaramj ?

Reply:
The syntax should be:
```super(GenerateTestRows, self).__init__()```